rfcs/012-merge-rebase-strategy.md

```diff
@@ -0,0 +1,34 @@
+# RFC 012 - Merge Settings/Merge Strategies
+
+## Affected projects
+
+All the projects under the [Open Collective organization](https://github.com/opencollective).
+
+## Motivation
+
+It's not always clear what merge strategy to use when merging PRs. Currently, core contributors are free to use
+whatever they prefer when merging PRs. GitHub allows three ways,
+
+1) Create a merge commit - All commits from the branch will be added  to the base branch via a merge commit.
+
+2) Squash and merge - All commits will be combined into one commit and added to the base branch.
+
+3) Rebase and merge - All the commits will be rebased and added to the base branch.
+
+This RFC proposes a solution in terms of GitHub settings so that we stick to one method.
```

Comment:
The `Motivation` section should explain the problems with the way we currently work, and what the proposed solution will improve. Rational arguments must always back the enforcement of such practices.

Reply:
@Betree : Ah, thanks for pointing it out. I've just added some more details on this. Hope that gives some more clarification. 😄 🐨 Let me know otherwise.